calculator/operations.py

```diff
@@ -1,19 +1,15 @@
 
```

Comment:
Your code in this file is what is causing your tests to fail under python3. In py3 the `reduce` function has been moved to the `functools` package. Thus you need to put a `from functools import reduce` at the top of your file.